ha-singleton-service/README.md

```diff
@@ -3,44 +3,47 @@
 
-For more information about singleton deployments, see _HA Singleton Deployments_ in the [Development Guide](https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/) for JBoss EAP.
+Be sure to inspect the `activate()` method of the `ServiceActivator` class for each example. Although the default election policy is used to build the singleton services for each of these examples, scripts and instructions are provided later in this document to demonstrate how to [configure other election policies](#configuring-election-policies).
+
+These examples are built and packaged as JAR archives.
+
+For more information about clustered singleton services, see _HA Singleton Service_ in the [Development Guide](https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/) for JBoss EAP.
 
 ## System Requirements
 
-The deployments this project produces is designed to be run on ${product.name.full} ${product.version} or later.
+The deployments this project produces are designed to be run on ${product.name.full} ${product.version} or later.
 
 Everything needed to build this project is ${build.requirements}. See [Configure Maven for ${product.name} ${product.version}](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/CONFIGURE_MAVEN_JBOSS_EAP7.md#configure-maven-to-build-and-deploy-the-quickstarts) to make sure the environment is configured correctly for testing the quickstarts.
 
 
-## Use of ${jboss.home.name}
+## Use of ${jboss.home.name}-1 and ${jboss.home.name}-2
+
+This quickstart requires that you clone your `${jboss.home.name}` installation directory and run two servers. The installation path is described in detail here: [Use of ${jboss.home.name} and JBOSS_HOME Variables](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/USE_OF_${jboss.home.name}.md#use-of-eap_home-and-jboss_home-variables).
 
-In the following instructions, replace `${jboss.home.name}` with the actual path to your ${product.name} installation. The installation path is described in detail here: [Use of ${jboss.home.name} and JBOSS_HOME Variables](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/USE_OF_${jboss.home.name}.md#use-of-eap_home-and-jboss_home-variables).
+In the following instructions, replace `${jboss.home.name}-1` with the path to your first ${product.name} server and replace `${jboss.home.name}-2` with the path to your second cloned ${product.name} server.
 
+## Clone the ${product.name} Directory
 
-## Setting Up the Test Environment
+While you can run this example starting only one instance of the server, if you want to see the singleton behavior, you must start at least two instances of the server. Copy of the entire ${product.name} directory to a new location to use for the second cluster member.
```

Comment:
We are supposed to use active voice in our docs, so I changed it from passive voice, "it can be run" , to active voice, "you can run".

See the explanation here: https://writing.wisc.edu/Handbook/CCS_activevoice.html

Good catch on the typo!